assets/enhancements.css

```
@@ -49,45 +49,613 @@
+  --line-strong:   #cccccc;
+
+  /* Code & Technical */
+  --code-bg:       #1f1f1f;
+  --code-bar:      #161616;
+  --code-line:     #2c2c2c;
+  --code-ink:      #ffffff;
+  --code-faint:    #888888;
+
+  /* Components */
+  --surface-card:  #ffffff;
+  --surface-card-hover: #f5f5f5;
```

Stylelint (17.14.0)

[error] Duplicate custom property "--surface-card-hover" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)